Modules/_posixsubprocess.c

```diff
@@ -20,6 +20,11 @@
 #ifdef HAVE_DIRENT_H
 #include <dirent.h>
 #endif
+#ifdef HAVE_GRP_H
+#include <grp.h>
+#endif /* HAVE_GRP_H */
+
+#include <limits.h>
```

Comment:
This is not needed, it's included in `Python.h` (and `pyport.h` contain workarounds in case some constants are missing).

Reply:
Removed.